docs/installation/linux.md

```diff
@@ -4,12 +4,14 @@
     You can check your Linux Kernel version with `uname -a`.
 
 Make sure that your OS (Operating System) is up-to-date.
-We recommend recent LTS releases such as **Ubuntu 18.04** or **Debian 10** (buster), minimal required versions:
+We recommend recent LTS releases such as **Ubuntu 20.04** or **Debian 10** (buster), minimal required versions:
 
 - Debian 9+
-- Ubuntu 16.10+
+- Ubuntu 18.04+
 - CentOS 7+
 
+*Note: While it might be possible to use older versions particular cases, we can only provide support for the above mentioned ones.*
```

Comment:
```suggestion
*Note: While it might be possible to use older versions in particular cases, we can only provide support for the above mentioned ones.*
```